### PR TITLE
The last pull request (#34) did not activate the *Publish to PyPi* workflow

### PR DESCRIPTION
**Pull request by LuisOlivaresJ**
Hi,

I am suspecting that the Action did not work because the **main** branch name. Additionally, we need to change the version on every pull request to OMG_master, otherwise packaging to PyPi will show an error because the version number already exist.

Cheers,
Luis

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ build-backend = "hatchling.build"
 [project]
 name = "omg_dosimetry"
 
-version = "1.5.5"
+version = "1.6.0"
```
Comment (petruong) [inline pyproject.toml:8]: So, from what I understand: is this the version line that has to be updated each pull request?

**Comment by petruong** [comment]
Would it be reasonable to include a few spelling error corrections in the README.rst for this pull request?
If so, I can add the changes in a commit prior to the merging of this pull request.

File: pyproject.toml (hunk 1 above)
Comment (jfcabana) [reply pyproject.toml:8]: Correct. Pull requests to Master should have the version number incremented. We're using the recommended "semantic versioning" (https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#semantic-versioning-preferred).

We implemented automatic actions that are triggered with every pull request to master. Documentation on ReadTheDocs is updated automatically, and the new version is built and published on pypi.

**Comment by jfcabana** [comment]
> Would it be reasonable to include a few spelling error corrections in the README.rst for this pull request? If so, I can add the changes in a commit prior to the merging of this pull request.

Why not!

**Comment by petruong** [comment]
> > Would it be reasonable to include a few spelling error corrections in the README.rst for this pull request? If so, I can add the changes in a commit prior to the merging of this pull request.
> 
> Why not!

Updated on my end.

**Review by jfcabana — APPROVED** [review]

**Comment by LuisOlivaresJ** [comment]
All right to me.

**Pull request merged**